ui/src/@types.tsx

```diff
@@ -32,7 +32,7 @@ export type RepoDataPropsT = {
   name: string
   createdAt: Date
   type: RepoType
-  automaticImport?: boolean
+  autoImportFrom?: string
```

Comment:
Is this API v1 already? If so, you might want to use a different field or establish some migration path as this would be a breaking change.

Reply:
mmmm understand.. but if am not wrong the purpose of previous field `automaticImport` is the same that the new one `autoImportFrom`, (know if repo was auto imported or not), in this case with this new one, if `autoImportFrom` is `null`, then repo was not auto imported. That could be make sense? @patrickdevivo might you confirm please?

Reply:
nothing is really "v1" yet as we're still pre-v1 in and under active development. This type represents a purely internal "api" for now as well, so it should be fine to change the type + name like so, only the UI is using it at the time being.

you are right though @JAORMX that in general and going forward "after v1" we should be careful about breaking changes to external facing fields (types and names) to minimize impact to downstream users/integrations and to communicate about it effectively. Thanks for keeping an eye out!